lib/utils/LinearAlgebra.hpp

```diff
@@ -43,7 +43,7 @@ template<typename MT, bool SO>
 double rms(const blaze::DenseMatrix<MT, SO>& A) {
     double t = 0.0;
 
-    auto At = ~A;
+    auto At = blaze::trans(A);
```

Comment:
I just want to flag @jrs65 on this change.  I remember we had tons of problems with the API change - and then the library version change.  I think this is safe as long as we keep the library version the same, but I don't remember all the details on what the issues were.